tests/Multicall.test.ts

```
@@ -0,0 +1,353 @@
+  let bob: Account;
+
+  let CHAIN_ID: string;
+
+  let sierraCode, casmCode;
+
+  const deployContracts = async () => {
+    try {
+      // Deploy HTLC
+      ({ sierraCode, casmCode } = await getCompiledCode("starknet_htlc_HTLC"));
+      htlc_data = new CallData(sierraCode.abi);
+      const constructor = htlc_data.compile("constructor", {
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)